src/hb-ucdn.cc

```diff
@@ -274,6 +274,9 @@ void free_static_ucdn_funcs (void)
 #endif
 
 extern "C" HB_INTERNAL
+hb_unicode_funcs_t *
+hb_ucdn_get_unicode_funcs (void);
```

Comment:
Not available on any header and clang doesn't like it

Reply:
Okay.  Not sure why I don't get the warning.

Reply:
The bot uses a brand clang compiler, so no one can come from Chrome project and say they need something :)